cmd/util/ledger/migrations/multiple_contract_migration.go

```diff
@@ -1,455 +0,0 @@
-package migrations
-
-import (
```

Comment:
I think is safer to keep the migrations around for future visibility

Reply:
Agreed, reverting

Reply:
Reverted in 5968363